README.md

```diff
@@ -10,11 +10,140 @@ several instruments from Mars orbiters are supported, but the system is
+Example call to ingest indices:
+
+```bash
+pdsc_ingest -c pdsc/config/hirise_rdr_metadata.yaml /PATH/TO/LBL/FOLDER /PATH/TO/OUT/FOLDER
+```
+
+Or, a mini example (I took a 9 row subset of the full table file and updated the index accordingly, and have added it to this git repository):
+
+```
+pdsc_ingest -c pdsc/config/hirise_rdr_metadata.yaml /PATH/TO/REPO/inputs_mini/hirise/RDRCUMINDEX.LBL /PATH/TO/OUT/FOLDER
+```
+
+# CTX example:
+
+Example call if you have access to analysis machines:
+
+```
+pdsc_ingest /pds/pdsfs2/pdsdata/mro/ctx/mrox_4098/index/cumindex.tab /PATH/TO/OUT/FOLDER
+```
+
+Or, a mini example (I took a subset of rows from the table file and updated the index accordingly):
+
+```
+pdsc_ingest /PATH/TO/REPO/inputs_mini/ctx/cumindex.tab /PATH/TO/OUT/FOLDER
+```
+
+
+# LROC
+
+LROC database:
+
+Example call if you have access to analysis machines:
+
+```
+pdsc_ingest -c pdsc/config/lroc_cdr_metadata.yaml /home/edunkel/PDS/lroc_proj/pdsc/from_pdsfs2/CUMINDEX.LBL /PATH/TO/OUT/FOLDER
+```
+
+Here is a mini example for lroc included in this repository:
+
+```
+pdsc_ingest -c pdsc/config/lroc_cdr_metadata.yaml /PATH/TO/REPO/inputs_mini/lroc/CUMINDEX.LBL /PATH/TO/OUT/FOLDER
+```
+
+
+# Basic Usage
+
+Follow the docs for basic usage, but make sure you're in the python prompt.
+
+## Example commands for LROC
+
+We show some example commands for LROC imagery below:
+
+```
+# set the directory where the database is stored (from the pdsc_ingest command):
+export PDSC_DATABASE_DIR=/PATH/TO/OUT/FOLDER
+python
+>>> import pdsc
+>>> pds_client = pdsc.PdsClient()
+>>> metadata = pds_client.query_by_observation_id('lroc_cdr', ' M101013931LC')
+>>> mydata = metadata[0]
+>>> localizer = pdsc.get_localizer(mydata, browse=False)
+>>> nrows = mydata.lines
+>>> ncols = mydata.samples
+>>> lat_center, lon_center = localizer.pixel_to_latlon(nrows//2, ncols//2)
+# The localizer will have an error on the order of 10-100s meters since we assume spherical
+>>> mydata.center_latitude
+>>> mydata.center_longitude
+# You can go back to pixel value
+>>> localizer.latlon_to_pixel(mydata.center_latitude, mydata.center_longitude)
+
+```
+
+# LROC Scripts
+
+To get lroc distributed samples around the lunar globe, you can call get_distritubed_samples.py in the scripts directory. Either pass in parameters from the command line, or update the defaults in the script. Here is a calling example:
+
+```
+conda activate p37
+# point to database
+export PDSC_DATABASE_DIR=/home/edunkel/PDS/lroc_proj/pdsc/outputs/lroc/ # example on JPL MLIA analysis machines
+python scripts/get_distributed_samples.py -o OUTPUT/FILE -n NUM_SAMPLES
+```
+
+This will print a list of examples with their sun angle (which helps with labeling, since craters look different depending on where the sun is).
+
+I have a script to assemble the data from this list in the deep learning repository here: https://github.jpl.nasa.gov/PDSIMG/deep-learning/blob/master/src/lroc/salience/scripts/assemble_data.py
+
+
```

Comment:
We might not want to include some of this in the public version of the repository, since it is specific to a JPL environment (similarly for references to JPL-specific projects). For any examples we do keep, we can probably re-write to exclude the `inputs` and `inputs_mini` directories to avoid distributing these files with the software itself.

Reply:
I agree with @garydoranjr . I don't think the Usage, Pytests, HiRISE Examples sections are necessary in the README file since they are already discussed in the PDSC Documentation.

Reply:
I removed from this repo and into the deep-learning repo here: (https://github.jpl.nasa.gov/PDSIMG/deep-learning/blob/master/auxiliary/pdsc_extras/README.md)